zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZkClientUriDomainMappingHelper.java

```diff
@@ -45,7 +51,7 @@
  *
  * The following illustrates the ZNode hierarchy:
  * . (root)
- * └── _CLIENT_URI_DOMAIN_MAPPING (mapping root path)
+ * └── /zookeeper/uri-domain-map (mapping root path)
```

Comment:
Should we reflect here that `uri-domain-map` is actually a configurable (and therefore variable) string?

Reply:
Sorry i didn't get you. It's not configurable anymore after my change right? @narendly

Reply:
Thanks for pointing it out - I missed it. If that's the case, I think it's good as is! Resolving.